src/app/router.js

```diff
@@ -64,6 +67,19 @@
         console.log('router Controller connected');
         self.closeSidenav();
       }
+
+      function calculate(){
+        // Это будет очень длинная функция. Пожалуйста, придумай что нибудь другое!
```

Comment:
Understood. I was stressed and in a hurry.
It is sometimes complicated to write clearly about development. Tell me in any case my comments are impossible to understand.